docs/prover/cli/options.md

```diff
@@ -37,7 +37,7 @@ When you wish to prove properties on the source code. This is by far the most co
 If we have a Solidity file `Bank.sol`, with a contract named `Bank` inside it, and a specification file called `Bank.spec`, the run command would be:  
 `certoraRun Bank.sol --verify Bank:Bank.spec`
 
-### `--assert`
+### `--assert_contract`
```

Comment:
isn't that assert_contracts? I'm always confusing this one

Reply:
There is assert_contracts but we marked is as "internal" so it should not be in this doc

Reply:
fixed (apparently  you are not the only one that get confused here)